src/posix/Makefile-posix

```diff
@@ -94,7 +94,10 @@ configure_OPTIONS                   = \
 # Platform specific switches
 
 ifeq ($(DAEMON),1)
-configure_OPTIONS              += --enable-posix-daemon
+configure_OPTIONS              += \
+    --enable-posix-daemon         \
+    LIBS=-lstdc++                 \
```

Comment:
It seems this adds dependency on gcc. Will this change cause issues on platforms without gcc toolchain?

Reply:
The stdc++ library is available for gcc and clang, on both ubuntu and macos.

Reply:
it seems stdc++ is not availabe on Android platform. see https://developer.android.com/ndk/guides/cpp-support

Reply:
@rxwen , any thoughts on this?

Reply:
I'm not quite sure about compiling with android ndk. Please wait me try it first.